### ISSUE TITLE
QA Ph3_Accessibility: Schedule - Add to calendar button not labeled in event detail 

### ISSUE THREAD

**Issue by brittasinnott** [GoogleChrome/ioweb2016#657]
With voice over enabled, the "add to calendar" button in event detail modals is simply labeled as "button", no further indication is given.  

Steps: 
1. Enable voice over in OS settings
2. Go to https://io-webapp-staging.appspot.com/io2016/
3. Select Schedule 
4. Select an event > detail modal expands 
5. Tab through options 
6. Note that the "add to calendar" button is labeled as "button" 

Chrome 49 Mac OS X 10.11

![gio_657](https://cloud.githubusercontent.com/assets/15128124/14468238/b147fa4a-0093-11e6-9aa6-bb59c78843ff.png)
